.github/workflows/manage-azure-policy.yml

```diff
@@ -42,8 +42,34 @@ jobs:
         wait: false
         scopes: |
           /subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb
-  
-  apply-azure-policy-live:
+
+  apply-azure-policy-subscription:
+    if: ${{ github.ref == 'refs/heads/master' }}
+    strategy:
+      matrix:
+        subscriptions: [2307d175-7e49-434b-9ac2-515529b845f2]
```

Comment:
I still don't understand why you need this

Reply:
I had meant to remove that. Updated.